### ISSUE TITLE
(tlg0019) EpiDoc and CTS conversion

### ISSUE THREAD

**Issue by lcerrato** [PerseusDL/canonical-greekLit#497]
created based on comments at end of https://github.com/PerseusDL/canonical-greekLit/issues/371

**Comment by lcerrato** [comment]
quoting @aurelberra

Hello. I may comment on this issue because I recently had the same problem in Aristophanes while setting up a little Capitains/Nemo [project](https://classnum-mnemosyne.herokuapp.com/) with my students.

Adding all line numbers and/or de-duplicating them is tricky. I tried to automate the processes with scripts (partly mine, partly Thibault Clérice's XSLT), but ended up doing quite a lot of manual corrections to get all lines numbered and unique (adding "b", "c", or "d" as needed). I always made sure the numbering inferred from `milestone`s remained valid (e.g. `<milestone ed="p" n="54" unit="card"/>` comes right before `<l n="54">`). However, every line in between those milestones can be misnumbered due to the mid-line speaker changes. As I worked through some of the files, I made educated guesses, but the numbers should be checked against the original editions (which not infrequently split and number the verses in an idiosyncratic way).

With this caveat, I think we have improved the files. The following editions have all their lines numbered:

tlg0019/tlg001/tlg0019.tlg001.perseus-grc1.xml
tlg0019/tlg002/tlg0019.tlg002.perseus-grc1.xml
tlg0019/tlg003/tlg0019.tlg003.perseus-grc2.xml
tlg0019/tlg004/tlg0019.tlg004.perseus-grc1.xml
tlg0019/tlg006/tlg0019.tlg006.perseus-grc2.xml
tlg0019/tlg007/tlg0019.tlg007.perseus-grc1.xml
tlg0019/tlg009/tlg0019.tlg009.perseus-grc2.xml
tlg0019/tlg010/tlg0019.tlg010.perseus-grc2.xml

In the other files the lines are still partially numbered:

tlg0019/tlg005/tlg0019.tlg005.perseus-grc1.xml
tlg0019/tlg008/tlg0019.tlg008.perseus-grc1.xml
tlg0019/tlg011/tlg0019.tlg011.perseus-grc1.xml

The files can be found at <https://github.com/classnum/mnemosyne/tree/master/corpora/data/tlg0019>.

I would be happy to know if you have a better system to deal with this problem, and whether the files should be contributed.
